src/promptflow/tests/sdk_cli_azure_test/recording_utilities/constants.py

```diff
@@ -41,17 +42,22 @@ class SanitizedValues:
     RESOURCE_GROUP_NAME = "00000"
     WORKSPACE_NAME = "00000"
     TENANT_ID = "00000000-0000-0000-0000-000000000000"
+    USER_OBJECT_ID = "00000000-0000-0000-0000-000000000000"
     # workspace
     DISCOVERY_URL = "https://eastus.api.azureml.ms/discovery"
     # datastore
     FAKE_KEY = "this is fake key"
     FAKE_ACCOUNT_NAME = "fake_account_name"
     FAKE_CONTAINER_NAME = "fake-container-name"
+    FAKE_FILE_SHARE_NAME = "fake-file-share-name"
     # aoai connection
     FAKE_API_BASE = "https://fake.openai.azure.com"
     # storage
     UPLOAD_HASH = "000000000000000000000000000000000000"
     BLOB_STORAGE_REQUEST_HOST = "fake_account_name.blob.core.windows.net"
+    # trick: "unknown_user" is the value when client fails to get username
+    #        use this value so that we don't do extra logic when replay
+    USERNAME = "unknown_user"
```

Comment:
what's the difference for remote operations between successfully getting user name and "unknown_user"?

Reply:
the only difference is the remote path prefix, for example:

- failed: `/Users/unknown_user/promptflow/flow_display_name/.gitattributes`
- successful: `/Users/zhengfeiwang/promptflow/flow_display_name/.gitattributes`